python-lib/dku_google/clusters.py

```diff
@@ -526,26 +603,44 @@ def __init__(self, project_id, zone, credentials=None):
             self.zone = default_zone
         else:
             self.zone = zone
+        if _is_none_or_blank(region):
+            default_region = self.zone.split("-")[:-1]
```

Comment:
```suggestion
            default_region = "-".join(self.zone.split("-")[:-1])
```
to fix that
```
Requesting cluster {
  "cluster": {
    "network": "https://www.googleapis.com/compute/v1/projects/dataiku-dev/global/networks/nmonville-vpc", 
    "description": "Created from plugin", 
    "addonsConfig": {
      "httpLoadBalancing": {
        "disabled": false
      }
    }, 
    "masterAuth": {
      "clientCertificateConfig": {
        "issueClientCertificate": true
      }
    }, 
    "name": "nmonville-cluster1", 
    "autopilot": {
      "enabled": true
    }, 
    "initialClusterVersion": null, 
    "ipAllocationPolicy": {
      "createSubnetwork": false, 
      "useIpAliases": true
    }, 
    "nodePools": [], 
    "subnetwork": "https://www.googleapis.com/compute/v1/projects/dataiku-dev/regions/europe-west1/subnetworks/nmonville-vpc-subnet", 
    "releaseChannel": {
      "channel": "STABLE"
    }, 
    "resourceLabels": {}
  }, 
  "parent": "projects/dataiku-dev/locations/[u'europe', u'west1']"
}
[2022-08-31 09:15:23,872] [5735/MainThread] [WARNING] [googleapiclient.http] Invalid JSON content from response: {
  "error": {
    "code": 403,
    "message": "Permission denied on 'locations/[u'europe', u'west1']' (or it may not exist).",
    "status": "PERMISSION_DENIED"
  }
}
Traceback (most recent call last):
  File "/opt/dataiku-dss-11.0.1/python/dataiku/cluster/server.py", line 50, in serve
    result = cluster.start()
  File "/tmp/tmp_folder_HQTDTJBY/dku_code.py", line 68, in start
  File "/data/dataiku/dss_data/plugins/installed/gke-clusters/python-lib/dku_google/clusters.py", line 355, in build
    raise Exception("Failed to create cluster : %s" % str(e))
Exception: Failed to create cluster : <HttpError 403 when requesting https://container.googleapis.com/v1/projects/dataiku-dev/locations/[u'europe',%20u'west1']/clusters?alt=json returned "Permission denied on 'locations/[u'europe', u'west1']' (or it may not exist).". Details: "Permission denied on 'locations/[u'europe', u'west1']' (or it may not exist).">
```